commit e196f999bb702662f25741507f9cf3245ec00189
Author: Florian Maurer <f.maurer@outlook.de>
Date:   Wed Dec 11 22:23:45 2024 +0100

    Revert "ipq40xx-generic: remove all dsl related packages"
    
    This reverts commit e5593d52ad3d459c6b8b5ab5d1b6be2f5ecb28f1.

diff --git a/targets/ipq40xx-generic b/targets/ipq40xx-generic
index 85509de7..669f7ad0 100644
--- a/targets/ipq40xx-generic
+++ b/targets/ipq40xx-generic
@@ -51,22 +51,6 @@ device('avm-fritz-box-4040', 'avm_fritzbox-4040', {
 device('avm-fritz-box-7530', 'avm_fritzbox-7530', {
 	factory = false,
 	aliases = {'avm-fritz-box-7520'},
-	packages = {
-		-- same as ATH10K_PACKAGES_IPQ40XX
-		'kmod-ath10k',
-		'-kmod-ath10k-ct',
-		'-kmod-ath10k-ct-smallbuffers',
-		'ath10k-firmware-qca4019',
-		'-ath10k-firmware-qca4019-ct',
-
-		-- VDSL modem
-		'-kmod-ltq-vdsl-vr11',
-		'-kmod-ltq-vdsl-vr11-mei',
-		'-ltq-vdsl-vr11-app',
-		'-ltq-dsl-base',
-		'-kmod-atm',
-		'-linux-atm',
-	},
 })
 
 device('avm-fritz-repeater-1200', 'avm_fritzrepeater-1200', {
